functions/Test-DbaValidLogin.ps1

```diff
@@ -281,42 +250,41 @@ Tests all logins excluding any that are from the subdomain Domain
 					continue
 				}
 				Write-Message -Message "Parsing Login $adlogin on $server" -Level Verbose
-				$exists = $enabled = $false
+				$exists = $false
 				if ($true)
 				{
 					$founduser = Get-DbaADObject -ADObject $adlogin -Type Group -Silent
 					if ($founduser) {
-						$exists = $enabled = $true
+						$exists = $true
 					}
 				}
 				else
 				{
 					Write-Warning -Message "AD Searcher Error for $groupname on $server" -Level Warning
 				}
-
-				if ($Detailed)
-				{
-					[PSCustomObject]@{
-						Server = $server.DomainInstanceName
-						Domain = $domain
-						Login = $groupname
-						Type = "Group"
-						Found = $exists
-						Enabled = $enabled
-						DisabledInSQLServer = $login.IsDisabled
-						ADLoginDetails = 'AD group'
-					}
+				$rtn = [PSCustomObject]@{
+					Server = $server.DomainInstanceName
+					Domain = $domain
+					Login = $groupname
+					Type = "Group"
+					Found = $exists
+					DisabledInSQLServer = $login.IsDisabled
+					AccountNotDelegated = $null
+					AllowReversiblePasswordEncryption  = $null
+					CannotChangePassword  = $null
+					PasswordExpired  = $null
+					Lockedout  = $null
+					Enabled  = $null
+					PasswordNeverExpires  = $null
+					PasswordNotRequired  = $null
+					SmartcardLogonRequired  = $null
+					TrustedForDelegation = $null
+					UserAccountControl = $null
 				}
-				else
-				{
-					[PSCustomObject]@{
-						Server = $server.DomainInstanceName
-						Domain = $domain
-						Login = $groupname
-						Type = "Group"
-						Found = $exists
-						Enabled = $enabled
-					}
+				if ($Detailed) {
+					Select-DefaultView -InputObject $rtn -ExcludeProperty UserAccountControl
+				} else {
+					Select-DefaultView -InputObject $rtn -ExcludeProperty UserAccountControl,AccountNotDelegated,AllowReversiblePasswordEncryption,CannotChangePassword,PasswordNeverExpires,SmartcardLogonRequired,TrustedForDelegation
```

Comment:
I would also like to rename the function itself, to make clear we check AD.
A login with sql authentication can be a 'valid login' too, so I think '..AD...' should be somewhere in the function name.
Do you have any suggestions?